services-api/src/main/java/io/scalecube/services/ServiceDefinition.java

```diff
@@ -0,0 +1,28 @@
+package io.scalecube.services;
+
+import java.util.Collections;
+import java.util.Map;
+
+/** Definition of service - type and tags. */
+public class ServiceDefinition {
```

Comment:
Call it `ServiceType`, service definition is too loud name for a tuple of Class+HashMap

Reply:
I don't think that's a sufficient argument. I see it as a service definition, perhaps in the future this entity will contain more information